homeassistant/components/combined_energy/coordinator.py

```diff
@@ -0,0 +1,101 @@
+"""DataUpdateCoordinator for the PVOutput integration."""
+from __future__ import annotations
+
+from abc import abstractmethod
+from datetime import timedelta
+from typing import TypeVar
+
+from combined_energy import CombinedEnergy
+from combined_energy.exceptions import CombinedEnergyAuthError, CombinedEnergyError
+from combined_energy.helpers import ReadingsIterator
+from combined_energy.models import DeviceReadings
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    LOG_SESSION_REFRESH_DELAY,
+    LOGGER,
+    READINGS_INCREMENT,
+    READINGS_INITIAL_DELTA,
+    READINGS_UPDATE_DELAY,
+)
+
+_T = TypeVar("_T")
+
+
+class CombinedEnergyCoordinator(DataUpdateCoordinator[_T]):
```

Comment:
```suggestion
class _CombinedEnergyCoordinator(DataUpdateCoordinator[_T]):
```

To make it protected

Reply:
Can I have feedback on why you did not implement the suggestion?

Reply:
Had a bad rebase, and had to unpick the mess a couple of changes where missed.